app/templates/apps/list.html

```diff
@@ -1,13 +1,35 @@
-{% extends "layouts/two-column.html" %}
+{% extends "layouts/one-column.html" %}
 
 {% set page_title = "Apps" %}
 
 {% block main_column %}
 
-<ul>
-  {% for app in apps.results %}
-  <li><a href="{{ url_for('apps.details', {id: app.id}) }}">{{ app.name }}</a></li>
-  {% endfor %}
-</ul>
+<p>{{ apps.results.length }} {{ "app" if apps.results.length == 1 else "apps" }}</p>
+
+<table class="list">
+  <thead>
+    <tr>
+      <th>ID</th>
+      <th>Name</th>
+      <th>Description</th>
+      <th>Repo</th>
+      <td>&nbsp;</td>
+    </tr>
+  </thead>
+  <tbody>
+    {% for app in apps.results %}
+    <tr>
+      <td>{{ app.id }}</td>
+      <td><a href="/apps/{{ app.id }}">{{ app.name }}</a></td>
```

Comment:
This href should be `{{ url_for('apps.details', {id: app.id}) }}`

Reply:
Ah, nice.